### ISSUE TITLE
Can't find variable: DOMException (React Native)

### ISSUE THREAD

**Issue by ibrahimchraibi** [Unleash/unleash-proxy-client-js#221]
### Describe the bug

Hello :)

We are currently using `@unleash/proxy-client-react` & `unleash-proxy-client` in our React Native project, according to the documentation React Native platform should be supported, but this exception started popping due to this line:

https://github.com/Unleash/unleash-proxy-client-js/blob/d13ad709213cad7bedd92c8a4c1bc9ee09d4a5c7/src/index.ts#L508

Which makes sense since `DOMException` is browser specific. 

Are we missing something? Wouldn't be enough to check for the exception name? 

Thank you very much for your answer and for the solid work on this awesome project!

### Logs, error output, etc.

```bash
ReferenceError <unknown>(unleash-proxy-client/build/main)
Level: Error
Can't find variable: DOMException
```

### Subscription type

Open source

### Hosting type

Self-hosted

### SDK information (language and version)

@unleash/proxy-client-react (React Native)

**Comment by kwasniew** [comment]
Hi @ibrahimchraibi ! Thanks for the report. We fixed this issue here: https://github.com/Unleash/unleash-proxy-client-js/pull/222
Next week we'll release a new version and bump up React SDK

**Comment by thomasheartman** [comment]
Hey, @ibrahimchraibi! We've released updated versions of both the JS SDK (v3.5.2) and the React SDK (4.3.1) now. If you're still seeing issues after that, let us know ☺️

**Comment by ibrahimchraibi** [comment]
Thanks a lot for the quick fix! :)
